apps/docs/layouts/guides/index.tsx

```diff
@@ -23,13 +23,14 @@ interface Props {
     tocVideo?: string
     canonical?: string
   }
+  editLink?: string
   children: any
   toc?: any
   currentPage?: string
   hideToc?: boolean
 }
 
-const Layout: FC<Props> = (props) => {
+const Layout: FC<Props> = (props, test) => {
```

Comment:
```suggestion
const Layout: FC<Props> = (props) => {
```

looks like this was kept by mistake?